css/scss/_documents.scss

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ * Document Pages
+ *
+ *
+*/
+.document {
+  header {
+    .doc-type {
+       i{
+         padding-right:10px;
+      }
+      span::after{
+        content: ', ';
+      }
+      span:last-child::after{
+        content: '';
+      }
+    }//.doc-type
```

Comment:
Are these closing comments helpful to anyone? I generally find them to be cluttery and inconsistently applied.